src/rust/bitbox02-rust-c/src/util.rs

```diff
@@ -29,9 +29,9 @@ pub extern "C" fn rust_util_all_ascii(cstr: CStr) -> bool {
 /// * `out_ptr` - Must be a valid pointer to an array of bytes that is buf_len*2+1 long
 #[no_mangle]
 pub extern "C" fn rust_util_uint8_to_hex(buf: Bytes, mut out: CStrMut) {
-    let out_len = out.as_ref().len();
```

Comment:
please update the `UNSAFE` notice, and the function documentation as well (I forgot that earlier).

Reply:
Done.